stdlib/http/src/main/ballerina/src/http/Module.md

```diff
@@ -1,5 +1,3 @@
-## Module overview
-
```

Comment:
Shouldn't we need to add relevant BBE links at the end of each section? HTTP and WebSocket?

Reply:
We already have ryt?